tests/python/mkl/test_quantization_mkldnn.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import sys
+import mxnet as mx
+
+os.environ['USE_MKLDNN'] = '1'

```

Comment:
Please rename it to 'ENABLE_MKLDNN_QUANTIZATION_TEST'

Reply:
Ideally, we'd gather this information from the backend rather than using this hack.

Reply:
@marcoabreu  Thanks for helping review. Agree that ideally we should have an API (from C level and python level) to differentiate various backend, we can revisit this after we have such API. And I will rename it to ENABLE_MKLDNN_QUANTIZATION_TEST for now.

Reply:
Thanks